src/app/pages/Bookmarks/Bookmarks.tsx

```diff
@@ -138,11 +17,16 @@ export default function Bookmarks(): JSX.Element {
         </Header>
       </section>
       <main className={styles.cardWrapper}>
-        {mockupData.length !== 0 &&
-          movieInfo.map((movie) => (
+        {errorMessage && <p>{errorMessage}</p>}
+        {!errorMessage && isLoading && <p>Loading...</p>}
+        {bookmarkedMovies &&
+          bookmarkedMovies.length > 0 &&
+          bookmarkedMovies.map((movie) => (
```

Comment:
This was only changed from `bookmarkedMovies.length !== 0 &&` because `bookmarkedMovies` will at some point contain an object `{ error: "Movie not found"}`, which leads to  `bookmarkedMovies` being `undefined`, which leads to `true`. (`undefined` in not `0` ).